webhosting/classic/how-to/configure-outlook.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: How to configure Outlook to receive and send email
+description: This page shows you how to Outlook to receive and send email
+dates:
+  validation: 2021-05-26
+  posted: 2021-05-26
+---
+
+You can read and send your emails using the Microsoft Outlook client. 
+<Message type="requirement">
+
+  - You have an account and are logged into the [Dedibox Console](https://console.online.net)
+  - You have created a webhosting **Classic**
```

Comment:
maybe add a link?
```suggestion
  - You have created a **Classic** webhosting 
```

Reply:
- No link possible here - the offer is EoL and no longer available